frontend/homeTabs.jsx

```diff
@@ -76,12 +77,20 @@ export function HomeTab({ navigation }) {
             corresponding route is a Stack.Screen in the parent
             Navigator (in App.js), the bottom tab bar is not displayed
             */}
-      <CardView home>
-        <LinearGradient colors={['#F9F6FF', '#CFDFF7']} style={{ padding: 20, paddingTop: 8, borderRadius: 10 }}>
-          <ButtonText>Last Night,</ButtonText>
-          <CardTitle center>2432 minutes</CardTitle>
-        </LinearGradient>
-      </CardView>
+      <HomeCardView>
+        <CardText>Last Night,</CardText>
+        <HomeCardTitle center data>{`${Math.floor(latestData.duration / 60)}hrs${latestData.duration % 60}min`}</HomeCardTitle>
```

Comment:
Do we want there to be no spaces anywhere in the HomeCardTitle?

<img width="478" alt="image" src="https://user-images.githubusercontent.com/3270324/105553265-e96ef400-5cb9-11eb-9e1c-89ca1c9bbea7.png">